migrations/1587950739709_challenge-data.js

```diff
@@ -0,0 +1,16 @@
+exports.up = function (pgm) {
+  pgm.createTable('challenges', {
+    id: { type: 'uuid', primaryKey: true },
+    name: { type: 'string', unique: true, notNull: true },
+    description: { type: 'string' },
+    files: { type: 'text[]' },
+    author: { type: 'string' },
+    category: { type: 'string' },
+    points: { type: 'integer', notNull: true },
+    flag: { type: 'string', notNull: true }
```

Comment:
We may want to store the challenge data as a JSON blob in the database so that we can also support additional challenge metadata that rCTF treats as opaque that other tools can utilize